fhir-persistence-jdbc/src/main/java/com/ibm/fhir/persistence/jdbc/postgres/PostgresResourceDAO.java

```diff
@@ -78,25 +77,14 @@ public Resource insert(Resource resource, List<ExtractedParameterValue> paramete
         final Connection connection = getConnection(); // do not close
         CallableStatement stmt = null;
         String stmtString = null;
-        Integer resourceTypeId;
         Timestamp lastUpdated;
-        boolean acquiredFromCache;
         long dbCallStartTime;
         double dbCallDuration;
 
         try {
-            resourceTypeId = getResourceTypeIdFromCaches(resource.getResourceType());
-            if (resourceTypeId == null) {
-                acquiredFromCache = false;
-                resourceTypeId = this.readResourceTypeId(resource.getResourceType());
-                this.addResourceTypeCacheCandidate(resource.getResourceType(), resourceTypeId);
-            } else {
-                acquiredFromCache = true;
-            }
-            if (logger.isLoggable(Level.FINER)) {
-                logger.finer("resourceType=" + resource.getResourceType() + "  resourceTypeId=" + resourceTypeId +
-                         "  acquiredFromCache=" + acquiredFromCache + "  tenantDatastoreCacheName=" + ResourceTypesCache.getCacheNameForTenantDatastore());
-            }
+            // Just make sure his resource type is known to the database before we
+            // hit the procedure
+            getResourceTypeId(resource.getResourceType());
```

Comment:
same comment as before getResourceTypeId is really not returning the id.  It's not as readable as it could be. something to think about for this pattern of code.

Reply:
same fix as above